### ISSUE TITLE
Do a checkup on the ActiveMQ and Karaf servers

### ISSUE THREAD

**Issue by bseeger** [jhu-idc/iDC-general#236]
Islandora relies heavily on passing messages around (outside of Drupal) to get back end services, like the image derivative service, to do their thing.  There is an ActiveMQ server sitting out there collecting messages that these services then grab. 

We've decided not to use fedora and have shut off a few of these back-end services. This ticket is to remind use to check in on the ActiveMQ and Karaf (alpaca) setup in Islandora to make sure that messages with no consumers (like the shut off services) are not building up in some ActiveMQ queue. 

This seems like a task to do once the system setup settles out and we have a functioning dev environment that is fairly stable.

**Comment by noahwsmith** [comment]
I recommend @dannylamb for this task

**Comment by bseeger** [comment]
I spent some time exposing the web console and created this [PR](https://github.com/jhu-idc/idc-isle-dc/pull/263) as a proof of concept to show how to get access to the web console again (using the containers created by this [PR](https://github.com/jhu-idc/idc-isle-buildkit/pull/81) (thanks, @birkland!)). 

The general ActiveMQ behavior I've observed is that upon system start up, 4 queues are created:

![Screen Shot 2021-10-27 at 4 50 20 PM](https://user-images.githubusercontent.com/7966375/139288370-bea58bc1-59d6-44df-93e1-ef4ec5a21e5d.png)

These were created when the consumers registered their interest in the queues.  
All created queues have registered consumers, so all messages, in theory, will get consumed, unless there is an issue with a consumer service slowing down or failing.

I tested out all of our ingests that included media (except for large file migrations tests) and am seeing messages getting enqueued and dequeued successfully, with nothing left on the queues. 

I observed that there are no queues created for unused services, like fedora, so no messages pile up unconsumed.

All in all, it seems like things checkout okay and no messages pile up in ActiveMQ for services we've turned off. 

Ideally we'd run a stress test for this, but I wasn't able to get an ingest from Michelle to try this one -- I will do that when she's around.
